lib/routes.jsx

```diff
@@ -115,6 +115,8 @@ var routeElements = Object.keys(pages).map(function(path) {
   return <Route path={path} component={pages[path]} key={path}/>;
 });
 
+var makerPartyLocales = ['de', 'fr', 'es', 'pl', 'it', 'nl', 'cs', 'bg'];
```

Comment:
@ScottDowne in learning, we've got some logic that picks the language based on user requested locale. If they request es-XX but only es is available, it'll return that.